testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/cacheddc/CachedDcDomainTestCase.java

```diff
@@ -57,6 +57,7 @@
 
     private static final long TIMEOUT_S = TimeoutUtil.adjust(30);
     private static final long NOT_STARTED_TIMEOUT_S = TimeoutUtil.adjust(7);
+    private static final int TIMEOUT_SLEEP_MILLIS = 300;
```

Comment:
This seems pretty long for a polling sleep. Better than 500 ms, but still pretty long. Maybe 25 or 50? WDYT? I don't think invoking ops that often will add much load to the server, but cutting 250ms off a poll loop a few times will add up.